### ISSUE TITLE
Downloading specific version gets you previous version's source

### ISSUE THREAD

**Issue by guidojw** [rhysd/actionlint#175]
When pinning actionlint to a version, for example v1.6.15:

```sh
bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/v1.6.15/scripts/download-actionlint.bash)
```

The actual downloaded version is v1.6.14.
I want to pin the version of actionlint instead of using the main branch to avoid getting broken workflow runs because of bugs introduced in actionlint (what happened with step level env context in v1.6.14).

**Comment by rhysd** [comment]
No, what you should run is:

```sh
bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash) 1.6.15
```
